src/components/Snapshot/SnapshotDownloadModal.tsx

```diff
@@ -68,38 +71,43 @@ const SnapshotDownloadModal: FC<
 
   return (
     <LightMode>
-      <Modal {...props}>
+      <Modal {...props} closeOnOverlayClick={false}>
         <ModalOverlay background="rgba(0,0,0,0.75)" />
         <ModalContent p="4rem" minW="40rem" color={NAMED_COLORS.DEEP_BLUE}>
           <ModalHeader>
-            <chakra.h2>Download Snapshot</chakra.h2>
+            <chakra.h2>Syncing your chain</chakra.h2>
           </ModalHeader>
-          <ModalCloseButton
-            color={NAMED_COLORS.GREY}
-            borderRadius="50%"
-            borderColor={NAMED_COLORS.LIGHT_GREY}
-            border="0.0125rem solid"
-            mt="1.5rem"
-            mr="1.5rem"
-          />
           <ModalBody>
             <chakra.h4>
-              You need to download our chain snapshot as the normal download
-              time could take up to {estimateTime}. The snapshot will be 2 times
-              faster and only {size}.{' '}
-              <Link display="none">If you need help, please click here.</Link>
+              Choose how to sync your app with the blockchain: <br />
+              <br /> Download Snapshot: Fast and centralized. Get a complete
+              copy quickly from a central source. This method ensures efficiency
+              and consistent data. <br /> <br />
+              Sync from Peers: Slower but decentralized. Retrieve the blockchain
+              from other users, contributing to network decentralization. While
+              it may take longer, it strengthens the network's resilience.{' '}
             </chakra.h4>
             {error && <chakra.h4 color={NAMED_COLORS.RED}>{error}</chakra.h4>}
           </ModalBody>
           <ModalFooter justifyContent="flex-start">
             <Button
+              style={{ marginRight: '16px' }}
```

Comment:
nit(non-blocking): Just FYI, because most of the `ui-kit` component are just Chakra components, they all take a bunch of style props by default. E.g. you should be able to do `marginRight="16px"` (or `mr="16px"` for a shorthand)  for this.

There's a ton of supported css properties like:

```
<Button
  marginLeft="1rem"
  marginRight="1rem"
  marginX="1rem" // left and right
  marginY="1rem" // top and bottom
  margin="1rem" // all 4 sides
  fontSize="1rem"
  border="1px solid red"
  whiteSpace="nowrap"
  position="relative"
>
  Yo
</Button>
```